.github/workflows/functional-test.yaml

```diff
@@ -533,10 +566,19 @@ jobs:
         with:
           name: ${{ matrix.name }}_container_logs
           path: ./${{ env.RADIUS_CONTAINER_LOG_BASE }}
+      - name: Delete azure resource group - ${{ env.AZURE_TEST_RESOURCE_GROUP }}
```

Comment:
Why is this changing as part of this PR? If we're just adding integration with checks, I wouldn't expect to see this change.